hazelcast/src/main/java/com/hazelcast/jet/impl/submitjob/memberside/JobUploadStatus.java

```diff
@@ -181,6 +181,20 @@ void createNewTemporaryFile() throws IOException {
         jobMetaDataParameterObject.setJarPath(jarPath);
     }
 
+    @SuppressWarnings("java:S5443")
+    Path getJarPath() throws IOException {
```

Comment:
nit: shouldn't it rather be `createJarPath`?

Reply:
rev : 3ea885e6b6dae7242783d4e962838ac3ae83523f